modes/simple-mpc/evil-collection-simple-mpc.el

```diff
@@ -0,0 +1,115 @@
+;;; evil-collection-mpc.el --- Bindings for `simple-mpc' -*- lexical-binding: t -*-
```

Comment:
`evil-collection-mpc.el` -> `evil-collection-simple-mpc.el`

Reply:
Oops! Fixed, thank you.